lib/testmap.py

```diff
@@ -127,6 +130,7 @@ def contexts(image, *scenarios: Iterable[str]):
         # These can be triggered manually with bots/tests-trigger
         '_manual': [
             'rhel-8-6/osbuild-composer',
+            'fedora-39',
```

Comment:
Lorax isn't a fedora thing, so that's overzealous. But it doesn't really hurt, of course.